e2e/multi-app-project/front-end/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/julienschmidt/httprouter"
+)
+
+// HealthCheck just returns true if the service is up.
+func HealthCheck(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
+	log.Println("🚑 healthcheck ok!")
+	w.WriteHeader(http.StatusOK)
+}
+
+// SimpleGet just returns true no matter what
+func SimpleGet(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
+	log.Println("Get Succeeded")
+	w.WriteHeader(http.StatusOK)
+	w.Write([]byte("front-end"))
+}
+
+// ServiceDiscoveryGet calls the back-end service, via service-discovery.
+// This call should succeed and return the value from the backend service.
+// This test assumes the backend app is called "back-end". The 'service-discovery' endpoint
+// of the back-end service is unreachable from the LB, so the only way to get it is
+// through service discovery. The response should be `back-end-service-discovery`
+func ServiceDiscoveryGet(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
+	endpoint := fmt.Sprintf("http://back-end.%s.local/service-discovery/", os.Getenv("ECS_CLI_PROJECT_NAME"))
```

Comment:
I think it might make sense. Setting it as SEARCH_DOMAIN seems so sad since it doesn't do exactly what a SEARCH_DOMAIN does. Maybe ECS_CLI_LOCAL_ENDPOINT?